src/angular/projects/spark-angular/src/lib/components/sprk-pagination/sprk-pagination.component.ts

```diff
@@ -315,6 +216,12 @@ export class SprkPaginationComponent {
     return this.currentPage === this.totalPages();
   }
 
+  // Returns true if the component is in either the default or long variant
+  // These variants render the same now
+  showNumbers(): boolean {
+    return this.paginationType === 'default' || this.paginationType === 'long';
```

Comment:
This could be more contextual by making it:
`this.paginationType !== 'pager'`

Reply:
I like this better than what I have. If the dev puts in an invalid paginationType I would want it to fall back to default. With what I have, it will fall back to Pager.